src/xcode/ENA/ENA/Source/Coordinator.swift

```diff
@@ -111,6 +111,8 @@ class Coordinator: RequiresAppDependencies {
 			],
 			animated: false
 		)
+		
```

Comment:
Maybe tell why we are nilling it:

```suggestion

		// prevents a loop
```

Reply:
added comment